domain/src/test/java/org/oppia/android/domain/oppialogger/analytics/PerformanceMetricsLoggerTest.kt

```diff
@@ -203,6 +213,15 @@ class PerformanceMetricsLoggerTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  private fun getStorageUsage(): Long {
+    val application = RuntimeEnvironment.application
+    val persistentUsage =
+      application.filesDir.totalSpace - application.filesDir.freeSpace
+    val cacheUsage =
+      application.cacheDir.totalSpace - application.cacheDir.freeSpace
+    return persistentUsage + cacheUsage
```

Comment:
I've introduced a fake to test this behaviour. 
Also to avoid confusion in further review since a similar test exists in the MetricUtils, we can't determine the exact value straightaway (even for tests) and we'll have to compute it one way or another. I've however mocked Context to return a test file whenever cacheDir or filesDir is called and then calculated their space values to verify the behaviour. I think that's a good enough way to verify api calling and value returning.